### DISCUSSION TITLE
api for cancelling, pause the subscription product order

### DISCUSSION THREAD

**Discussion by manoharsinghrawat-cedcoss** [Shopify/storefront-api-feedback#100] [Help] [answered]
Is there any api available to pause or cancel the subscription product order?

**Comment by johncraigcole** [comment, marked as answer]
The Admin API does have the ability to modify a given subscription contract. Here's the Subscriptions API info form the [developer docs](https://shopify.dev/apps/subscriptions). Be aware that the ability to read and modify subscriptions are restricted to the app that created them. Put another way, one subscription app can't read or modify the contract created by another app.
